src/ethereum/prague/vm/__init__.py

```diff
@@ -131,6 +131,7 @@ class Message:
     is_static: bool
     accessed_addresses: Set[Address]
     accessed_storage_keys: Set[Tuple[Address, Bytes32]]
+    is_delegated: bool
```

Comment:
The name of this variable is ambiguous. Is it for `DELEGATECALL` or EIP-7702 or both? Perhaps something like `precompile_behavior: PrecompileBehavior` where:

```python
class PrecompileBehavior(Enum):
    STOP_IMMEDIATELY
    EXECUTE
```

(names to be bikeshedded)

Reply:
Updated. Changed the variable name to `disable_precompiles`